2019-10-17-RobertThomas/index.md

```diff
@@ -0,0 +1,31 @@
+
+This article was interesting to me for a number of reasons. One being that you don't hear many cases today of young men trying to get with older women for their money, most of the time it's the other way around with young women trying to find that rich lonely man to latch onto. Another thing I found interesting was how the family was able to literally nullify the marriage without the consent of either the bride or the groom and the fact that it was completely legal. There has been ongoing debate and trials regarding ways to legally nullify marriages in the U.S as outlined [here](http://eds.b.ebscohost.com/eds/pdfviewer/pdfviewer?vid=5&sid=ed4cddda-385a-463f-9405-36c8c02418ee%40pdc-v-sessmgr01), though I know of no way in America for someone who has no part in the marriage to nullify it against the bride and groom's wishes so the fact that this was allowed amazes me. However [here](http://eds.b.ebscohost.com/eds/pdfviewer/pdfviewer?vid=2&sid=ed4cddda-385a-463f-9405-36c8c02418ee%40pdc-v-sessmgr01) it is shown how in India a marriage may be nullified for a variety of reasons including if either the bride or groom was declared mentally unsound. This could be attributed to the laws in India that require a woman to have a male guardian so they are seen as less than women are seen in America. I read an article [here](https://www.indiatoday.in/india/story/kerala-high-court-hindu-muslim-islamic-state-979157-2017-05-25) on how a marriage was made null and void by a court in India over the fact that it was believed the daughter was forcibly converted to Islam. Apparently this isn't extremely uncommon for countries other than the U.S.
+
+Interestingly enough I found a blog post [here](https://dig-eg-gaz.github.io/post/18-blog-pustam/) that decided to write about an article that was extremely upset with the fact that marriages are a way for women to be happy. I found it ironic considering Mrs. Waghida's family forcibly put an end to her marriage even though she inferably was happy. You could say that her family was trying to prevent her from getting her heart broken by this swindler or just hurt in general because as outlined [here](https://dig-eg-gaz.github.io/post/18-blog-donofrio/) where a man attempting to fix his relationship with his wife ends up stabbing her out of anger, domestic violence was not uncommon.
```

Comment:
Tune up your first sentence. The blog post was not the one that "decided", and the article was not the one that was "extremely upset."